src/components/plots/PlotContainer.jsx

```diff
@@ -36,42 +52,51 @@ const PlotContainer = (props) => {
       return currentConfig[key] === initialConfig[key];
     });
 
-    return isConfigEqual;
+    return isEqual;
+  };
+
+  const handleResize = () => {
+    const direction = window.innerWidth > 1024 ? 'row' : 'column';
+    if (tileDirection !== direction) setTileDirection(direction);
   };
 
+  useEffect(() => {
+    window.addEventListener('resize', handleResize);
+  }, []);
+
   useEffect(() => {
     if (!config) {
       return;
     }
 
-    if (_.isEqualWith(config, initialPlotConfigStates[plotType], checkConfigEquality)) {
-      setEnableReset(false);
+    if (isConfigEqual(config, initialPlotConfigStates[plotType])) {
+      setResetDisabled(true);
       return;
     }
 
-    setEnableReset(true);
+    setResetDisabled(false);
   }, [config]);
 
   const onClickReset = (event) => {
     event.stopPropagation();
     dispatch(resetPlotConfig(experimentId, plotUuid, plotType));
-    setEnableReset(false);
+    setResetDisabled(true);
   };
 
   if (!config) {
     return <Skeleton active paragraph={{ rows: 1 }} title={{ width: 500 }} />;
   }
 
-  const renderExtra = () => (
-    <Space>
-      {extra}
-      {!showReset ? (
+  const renderPlotToolbarControls = () => (
+    <Space style={{ marginRight: '0.5em' }}>
+      {extraToolbarControls}
+      {showReset ? (
         <Button
           key='reset'
           type='primary'
           size='small'
           onClick={onClickReset}
-          disabled={!enableReset}
+          disabled={resetDisabled}
         >
           Reset
```

Comment:
Can you use the ResetButton component you created here and remove all this code including the functions and variables working with it?

Reply:
The `ResetButton` was a carry over from a previous commit, I'm using this instead of that